waltz-server/src/main/java/com/wepay/waltz/server/WaltzServer.java

```diff
@@ -471,6 +472,7 @@ private void registerMetrics() {
         REGISTRY.gauge(metricsGroup, "endpoint", (Gauge<String>) () -> endpoint.toString());
         REGISTRY.gauge(metricsGroup, "waltz-server-num-partitions", (Gauge<Integer>) () -> getPartitionIds().size());
         REGISTRY.gauge(metricsGroup, "replica-info", (Gauge<Map<Integer, List<String>>>) () -> getReplicaInfoMap());
+        REGISTRY.gauge(metricsGroup, "waltz_version_id", (Gauge<String>) () -> WaltzInfoParser.getVersion());
```

Comment:
How about using hyphens to be consistent with other metric name?

Reply:
Good point. Fixed!